uint8 msgType
uint8 msgLength
uint8[] msgBody
int32 timeStamp
int16 error
